about/list_of_features.rst

```diff
@@ -663,6 +665,11 @@ XR support (AR and VR)
 
   - Including support for popular stand alone headsets like the Meta Quest 1/2/3 and Pro, Pico 4, Magic Leap 2, and Lynx R1.
 
+- Out of the box limited support for visionOS Apple headsets.
+
+  - Currently only exporting an application for use on a flat plane within the
+    headset is supported. Immersive experiences are not supported.
+ 
```

Comment:
Apple describes iPhone and iPad apps compiled for visionOS as _Compatible Apps_:

* [Create for visionOS](https://developer.apple.com/visionos/get-started/#create)

  > Note: People can also run your existing iPadOS or iOS app as a **compatible app** in visionOS. Your app appears as a single, scalable window in the person’s surroundings.
* [Making you app compatible with visionOS](https://developer.apple.com/documentation/visionos/making-your-app-compatible-with-visionos)

We could include these links to give developers more information.